File: Examples/src/main/java/com/aspose/words/examples/quickstart/LoadAndSaveToDisk.java
package com.aspose.words.examples.quickstart;

import com.aspose.words.Document;
import com.aspose.words.examples.Utils;

public class LoadAndSaveToDisk {
    public static void main(String[] args) throws Exception {
        //ExStart:LoadAndSaveToDisk
        // The path to the documents directory.
        String dataDir = Utils.getDataDir(LoadAndSaveToDisk.class);
        //ExStart:OpenFromFile
        // Load the document from the absolute path on disk.
        Document doc = new Document(dataDir + "Document.doc");
        //ExEnd:OpenFromFile
        // Save the dDocument.dococument as DOCX document.");
        doc.save(dataDir + "Document Out.docx");
        //ExEnd:LoadAndSaveToDisk
        System.out.println("Document loaded from disk and saved again successfully.");
    }
}

File: Plugins/Aspose_Words_Java_for_Jython/asposewords/quickstart/LoadAndSaveToDisk.py
from asposewords import Settings
from com.aspose.words import Document

class LoadAndSaveToDisk:

    def __init__(self):
        dataDir = Settings.dataDir + 'quickstart/'
        
        doc = Document(dataDir + 'Document.doc')
        
        doc.save(dataDir + 'Document_Out.doc')
        
        print "Document saved."

if __name__ == '__main__':           
    LoadAndSaveToDisk()